about.html

```diff
@@ -72,13 +72,26 @@ <h2>2023-19-5(Fri) <br><span class="span">CEO, Jalloh ~</span> StartUp TidCoin</
       <div class="image">
         <img class="mobile" src="images/image-mockups.png" alt="image-mockups" width="100%">
       </div>
+
     </section>
     <h3 class="parag1">Creative Commons history of Easybank and TidCoin</h3>
     <div class="section-creative-logo parag1">
       <p>Creative Commons history of Easybank and TidCoin - 2020 during the COVID-19 pandemic. Easybank and TidCoin emerged as two innovative fintech companies during the unprecedented times of the COVID-19 pandemic in 2020. As physical distancing measures were implemented globally, more and more people began relying on digital banking solutions for managing their finances. Easybank and TidCoin quickly adapted to this shift in behavior and saw tremendous growth in their user base during this period.</p>
   </div>
   <div class="logo-history parag1">
     <img src="images/logo.svg" alt="logo">
+  </div>
+  <div class="grids-web">
+    <div class="webinar">
+      <div class="container">
+        <p>2023</p>
+      </div>
+    </div>
+    <div class="webinar-2">
+      <div class="container">
+        <p>2024</p>
+      </div>
```

Comment:
- [ ] Great job working on some of the requested changes but I think there are some thing you should fix as required by the previous reviewer your about page should be identical to the given design since it is mentioned in the requirements , since you have some sections are different in your design, I will show the differences ⬇💯

![image](https://github.com/aradradev/easyBankTC/assets/58553711/1a465355-7d00-493d-85b8-3d78d063effe)

- Here you have the text for the heading and paragraph has a bg color but it shouldn't have it so kindly remove it
-  I could see the text should be aligned to the center with some margins in the left and right so kindly stick with the given design as much as possible ✨🤩

- Your desigm
![image](https://github.com/aradradev/easyBankTC/assets/58553711/6cd41a7c-2f1f-47df-beae-f2455722c630)

- Given
![image](https://github.com/aradradev/easyBankTC/assets/58553711/344a1c67-c596-4d13-9304-f27a20e8775b)